frontends/p4/fromv1.0/converters.h

```diff
@@ -78,12 +78,23 @@ class TypeConverter : public ExpressionConverter {
     explicit TypeConverter(ProgramStructure* structure) : ExpressionConverter(structure) {}
 };
 
+class ExternConverter {
+ public:
+    ProgramStructure *structure = nullptr;
+
+    virtual const IR::Type_Extern *convertExternType(const IR::Type_Extern *, cstring);
```

Comment:
Can you please Doxygen the parameters?
Or at least give a readable name to the cstring parameter?